content/200-orm/500-reference/250-error-reference.mdx

```diff
@@ -316,6 +320,18 @@ Possible P1012 error messages:
 
 "Transaction failed due to a write conflict or a deadlock. Please retry your transaction"
 
+#### <inlinecode>P2035</inlinecode>
+
+"Assertion violation on the database: {database_error}"
+
+#### <inlinecode>P2036</inlinecode>
+
+"Error in external connector (id {id})"
+
+#### <inlinecode>P2037</inlinecode>
+
+"Too many database connections opened"
```

Comment:
```suggestion
"Too many database connections opened: {message}"
```
A mistake like this should not be possible with the script. Are we not using this any more?

Reply:
I just copied over the descriptions from here: https://github.com/prisma/docs/issues/5639#issue-2136235549

About the script: I think I've heard that we have smth like this but I don't know where it is / how to use it. Do you have any details about it by any chance? (If not no worries, I'll do some research myself as well.)

Happy to update the PR with the actual output of the script once I figured out how to use it.